geniza/corpus/templates/corpus/snippets/document_option_label.html

```diff
@@ -0,0 +1,25 @@
+{# template snippet for displaying document label on a form #}
+{# used on document merge form to provide enough information to merge accurately #}
+<div class="merge-document-label">
+    <strong><a target="_blank" href="{% url 'admin:corpus_document_change' document.id %}">{{ document }}</a></strong>
+    <p>PGPID {{ document.id }}</p>
+    {% if document.description %}
+        <p>
+            <dt>Description</dt>
+            <dd>{{ document.description }}</dd>
+        </p>
+    {% endif %}
+
+    {% if document.footnotes.count %}<strong>Footnotes</strong>{% endif %}
```

Comment:
Let's rename to scholarship records instead of footnotes (not sure what h-level this should be)

```suggestion
    {% if document.footnotes.count %}<strong>Scholarship Records</strong>{% endif %}
```